metabase/metabase
# Add recents to the new homepage

Epic: https://github.com/metabase/metabase/issues/21322

How to test:
- Open a fresh Metabase instance
- Make sure that after the installation you see x-rays
- Create a question or a dashboard
- Make sure that now you see recent items on the homepage
- Ignore broken e2e tests for now

<img width="791" alt="Screenshot 2022-04-01 at 10 51 02" src="https://user-images.githubusercontent.com/8542534/161219594-2adf6c6a-eac7-48ff-bf09-f841bcd1f65d.png">
